vectordb_bench/backend/clients/elastic_cloud/elastic_cloud.py

```diff
@@ -145,7 +147,16 @@ def search_embedding(
 
     def ready_to_search(self):
         """ready_to_search will be called between insertion and search in performance cases."""
-        pass
+        assert self.client is not None, "should self.init() first"
+        self.client.indices.refresh(index=self.indice)
+        force_merge_task_id = self.client.indices.forcemerge(index=self.indice, max_num_segments=1, wait_for_completion=False)['task']
+        log.info(f"Elasticsearch force merge task id: {force_merge_task_id}")
+        SECONDS_WAITING_FOR_FORCE_MERGE_API_CALL_SEC = 30
+        while True:
+            time.sleep(SECONDS_WAITING_FOR_FORCE_MERGE_API_CALL_SEC)
+            task_status = self.client.tasks.get(task_id=force_merge_task_id)
```

Comment:
Any estimation how long it will take, since we are considering to introduce some timeout mechanism for all tasks.

Reply:
Many minutes